application/src/main/java/org/togetherjava/tjbot/commands/tag/TagUtility.java

```diff
@@ -0,0 +1,68 @@
+package org.togetherjava.tjbot.commands.tag;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Emoji;
+import net.dv8tion.jda.api.entities.MessageChannel;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.components.Button;
+import net.dv8tion.jda.api.interactions.components.ButtonStyle;
+import org.togetherjava.tjbot.tag.TagSystem;
+
+import java.awt.*;
+import java.time.LocalDateTime;
+
+public class TagUtility {
+    private TagUtility() {}
```

Comment:
if u want, u can use the enum-pattern `public enum TagUtility { ; ...}`

just minor though, i actually prefer it ur way cause an enum could distract a reader for only the minor benefit of adding security against serialization and reflection (not really worth the deal).